### PR TITLE
Create MagicSmokeIndustriesModelRework

### PR DESCRIPTION
**Pull request by Sheepingtin**
Magic Smoke Industries Infernal Robotics - Model Rework - Working with KSP 1.2.2

### CODE REVIEW

**Comment by LemonzPC** [comment]
Tested and working in game with my last edit to the main netkan file. The legacy pack has been delayed do to compatibility with ckan, Fixes are waiting for approval.

**Comment by politas** [comment]
I believe this should be depending on `"InfernalRobotics"` rather than `"MagicSmokeIndustries"`

File: NetKAN/MagicSmokeIndustriesModelRework.netkan [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "spec_version": 1,
+    "identifier"  : "MagicSmokeIndustriesModelRework",
+    "name"        : "Magic Smoke Industries Infernal Robotics - Model Rework",
+	"author"      : ["ZodiusInfuser","Ziw"],
+	"abstract"    : "More robotic parts!",
+    "version"     : "1.1.2",
+    "ksp_version" : "1.2.2",
+    "license"     : "GPL-3.0",
+	    "depends": [
+        { "name": "MagicSmokeIndustries" }
```
Comment (politas) [inline NetKAN/MagicSmokeIndustriesModelRework.netkan]: I believe this should be "InfernalRobotics" rather than "MagicSmokeIndustries"

**Comment by Dazpoet** [comment]
We do have these old ones laying about:
https://github.com/KSP-CKAN/NetKAN/blob/master/NetKAN/IR-Model-Rework-Core.frozen https://github.com/KSP-CKAN/NetKAN/blob/master/NetKAN/IR-Model-Rework-Expansion.frozen https://github.com/KSP-CKAN/NetKAN/blob/master/NetKAN/IR-Model-Rework-Utility.frozen
but seeing as this PR is only one file I guess the author might have dropped the old notation of splitting it into multiple parts?

**Comment by politas** [comment]
Ah, I think we need to look at changing those instead.

**Pull request closed without merging**